src/api/payment-requests/payment-requests.md

```diff
@@ -1070,6 +1071,50 @@ Confirmations require an `idempotencyKey` in order to ensure that a confirmation
 | 403    | {% break _ INVALID_AMOUNT   %}             | The confirmation is greater then the remaining funds on the authroization                                                                                           |
 | 403    | {% break _ IDEMPOTENT_OPERATION_FAILED %}  | There has already been a confirmation against the Payment Request with the same idempotencyKey but different content. We recommend that you void this confirmation. |
 
+<a name="void-confirmation"></a>
+### Void a confirmation against a Pre Auth Payment Request **EXPERIMENTAL**
+
+Voiding a confirmation will reverse the confirmation. An `idempotencyKey` is required as a reference to the confirmation to be voided.
```

Comment:
We should document a 200 here even when we don't know what the idempotency key means. So if a void-confirm comes through, we'd say something like "We've done everything we can with this in our system to back out any confirmation associated with that idempotency key".